xml_inscriptions/transcribed/MA.Bos.Gardn.G.SG8e8.xml

```diff
@@ -75,6 +75,13 @@
                      <date evidence="unspecified" notBefore="0060" notAfter="0090">60-90 CE</date>
 
                   </origin>
+                  <provenance type="observed" subtype="recorded">
+                     <p>Transcribed and described by <persName>Friedrich von Duhn</persName> (1851-1930) in the <placeName>Vinea Pacca, Rome</placeName> in <date when="1881">1881</date>.</p>
```

Comment:
Add `@role` to the `<persName>` here, if you have a chance.

Reply:
I wasn't sure what role to use here. Recorder?